auction-server/src/opportunity/entities/token.rs

```diff
@@ -36,3 +36,10 @@ impl From<api::TokenAmountSvm> for TokenAmountSvm {
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct TokenMint {
+    pub mint:     Pubkey,
+    pub decimals: u8,
+    pub owner:    Pubkey,
```

Comment:
i prefer `token_program` for clarity. technically owner is unambiguous, but there are some other terms that are close enough (the mint, freeze, etc. authorities) such that `token_program` is just clearer